src/pages/Spreadsheet.tsx

```diff
@@ -16,6 +16,29 @@ function spreadsheet(text = 'spreadsheet') {
   )
 }
 
+export const SubscribeForm: React.FunctionComponent = () => {
+  return (
+    <>
+      <h2>Subscribe for updates</h2>
+      <p>
+        We will send email updates when we make feature upgrades and bug fixes
+        to the spreadsheet.
+      </p>
+      <p>
+        <a
+          href="http://eepurl.com/hb6y4T"
+          target="_blank"
+          rel="noreferrer"
+          className="btn btn-secondary"
+          role="button"
```

Comment:
I find it weird that this looks like a button but is just a link. Elsewhere our buttons are buttons and our links are links. Could you make this consistent?